.gitignore

```diff
@@ -1,3 +1,4 @@
 coverage/
 dist/
 node_modules/
+.log/
```

Comment:
What is producing this `.log` folder?

Reply:
Emacs LSP plugin for JavaScript/TypeScript

Reply:
I prefer to keep all editor-specific stuff out of project-specific `.gitignore` files. That stuff should go in people's global `.gitignore`.

Would you mind removing?

Reply:
Removed. Thanks the feedback!